Software-Construction/Inheritance-And-Composition/ZooManagementSystem/Zoo.java

```diff
@@ -0,0 +1,139 @@
+package ZooManagementSystem;

```

Comment:
Use of name capacity for both zoo capacity and cage capacity is a bit confusing, the names could have been segregated like cageCapacity and zooCapacity